CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v3.2.2
+
+* If only `private_subnet_ids` are provided then the Fargate service and the ALB will be placed in them and will become an internal-facing load-balancer.
+* If only `public_subnet_ids` are provided then the Fargate sercice and the ALB will be placed in them and will become a public-facing load-balancer.
+* If both `private_subnet_ids` and `public_subnet_ids` are NOT provided then the default VPC subnets will be assumed and will become a public-facing load-balancer.
```

Comment:
Style suggestion: swap the "double-negative" line with the "double-positive" one below to let the affirmative go first, and rephrase to ```If neither  `private_subnet_ids` nor `public_subnet_ids` are provided...```